### PR TITLE
Add translations

### PR DESCRIPTION
**Pull request by knabo6**

### CODE REVIEW

File: src/app/_shared/_components/header/header.component.html [hunk 1]
```diff
@@ -1,4 +1,15 @@
 <div class="container text-center my-4">
-    <h1 class="display-4 font-monospace" style="font-variant: small-caps;">Ketal</h1>
-    <p class="lead">Grosse Guinze Generator</p>
+<form [formGroup]="languageForm">
+  <div class="row position-absolute top-0 end-12">
+    <label for="language-select" class="col-form-label col-5" [translate]="'Label_Language'"></label>
+    <div class="col-7">
```
Comment (knabo6) [inline src/app/_shared/_components/header/header.component.html]: col 7 to small 
image

File: src/app/_components/players/players-list/players-list.component.html [hunk 2]
```diff
@@ -2,17 +2,15 @@
...
-    <div *ngIf="newPlayer?.invalid && (newPlayer?.dirty || newPlayer?.touched)" class="alert alert-danger">
-      <div *ngIf="newPlayer?.errors?.['required']">
-        Le nom du joueur est requis.
-      </div>
+    <div *ngIf="(newPlayer?.invalid && (newPlayer?.dirty || newPlayer?.touched)) && newPlayer?.errors?.['required']" class="alert alert-danger">
+      <span [translate]="'Label_Error_NameRequired'"></span>
     </div>
-    <div *ngIf="this.playerHelper.getPlayerNumber() === 23"  class="alert alert-secondary">
+    <div *ngIf="this.playerHelper.getPlayerNumber() === 23" class="alert alert-secondary">
       Le nombre maximum de joueurs est atteint.
```
Comment (brboi) [inline src/app/_components/players/players-list/players-list.component.html]: not translated

**Review by brboi — CHANGES_REQUESTED** [review]
maybe there are other non translated terms

File: src/app/_shared/_components/header/header.component.html [hunk 3]
```diff
@@ -1,4 +1,15 @@
...
+      <select id="language-select form-select-sm" class="form-select" formControlName="selectedLanguage"
+        (change)="onLanguageChange()">
+        <option *ngFor="let lang of getPossibleLanguages()" [value]="lang.shortName"> {{lang.name}}</option>
+      </select>
+    </div>
+  </div>
+</form>
+    <h1 class="display-4 font-monospace" style="font-variant: small-caps;" [translate]="'Label_App_¨Title'"></h1>
```
Comment (brboi) [inline src/app/_shared/_components/header/header.component.html:12]: Why translate app title ?

File: src/app/app.component.html
Comment (brboi) [inline src/app/app.component.html:1]: this whole diff seems only linting

**Review by brboi — CHANGES_REQUESTED** [review]

File: src/app/_shared/_components/header/header.component.html (hunk 1 above)
Comment (Solacin21) [reply src/app/_shared/_components/header/header.component.html]: Changed, should be alright now

File: src/app/_shared/_components/header/header.component.html (hunk 3 above)
Comment (Solacin21) [reply src/app/_shared/_components/header/header.component.html:12]: Why not? :) Well, It is still an hardCoded string so, I add it to the list of translations

File: src/app/_components/players/players-list/players-list.component.html (hunk 2 above)
Comment (Solacin21) [reply src/app/_components/players/players-list/players-list.component.html]: Fixed

File: src/app/app.component.ts [hunk 4]
```diff
@@ -9,6 +11,12 @@ import {GameService} from "./services/game/game.service";
 export class AppComponent {
   title = 'ketal';
 
-  constructor(public gameSrv: GameService) {
+  constructor(
+    public gameSrv: GameService,
+    private translate: TranslateService
+  ) {
+    const defaultLang = environment.defaultLanguage;
```
Comment (lammensmichel) [inline src/app/app.component.ts]: maybe use this.translate.getBrowserLang()  ?

File: src/assets/i18n/fr.json [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+  "Label_Join": "Rejoindre",
+  "Label_CreateNewChannels": "Créer un Nouveau Salon",
+  "Label_ChannelsName": "Nom du Salon",
+  "Label_ChannelsDescription": "Description du Salon",
+
+  "Label_Error_NameRequired": "Le nom du buveur est requis.",
+  "Label_Error_TooMuchPlayer": "Le nombre maximum de joueurs est atteint.",
+
+  "Button_Add": "Ajouter",
+  "Button_BeginGame": "Débuter la Grosse Guize",
```
Comment (brboi) [inline src/assets/i18n/fr.json]: Typo: `Guinze`

**Pull request merged**